recipes/pom.xml

```diff
@@ -22,6 +22,15 @@
         <rewrite-maven-plugin.version>4.46.0</rewrite-maven-plugin.version>
         <!--   for now, we don't know where to find compatibility information for the Gradle plugin -->
         <rewrite-gradle-plugin.version>5.40.6</rewrite-gradle-plugin.version>
+        <!-- If version from rewrite-recipe-bom is used, error occures during testing ->
```

Comment:
Weird behavior is happening if the version from BOM is used for `rewrite-migrate-java` dependency. `NoSuchMethod 'com.fasterxml.jackson.core.util.TextBuffer com.fasterxml.jackson.core.io.IOContext.constructReadConstrainedTextBuffer()'` is thrown during execution of the tests. I forced version to 1.16.0 which does not cause the error. TBH I'm not completely sure why this is happening, BOM should contain compatible versions.